indico/htdocs/sass/partials/_tables.scss

```diff
@@ -36,9 +36,13 @@ table.i-table {
 tr.i-table {
     border-bottom: 1px solid #E5E5E5;
     border-top: 1px solid #E5E5E5;
+
+    &.selected {
+        background-color: #CDEB8B;
```

Comment:
It's not in `_palette.scss` and it's the color that was used in the old code. Do you think I should put it in the palette? I tried to use a different color from the palette but I didn't like any other...